### DISCUSSION TITLE
Κριτήρια βαθμολόγησης κουίζ

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/sw#1149] [Q&A] [answered]
Κριτήρια βαθμολογίας: 10% Συμμετοχικό εκπαιδευτικό περιεχόμενο + 10% κατασκευή του βιβλίου + 10% γραμμή εντολών + 30% ομαδικότητα και συνεργασία + 40% συμμετοχή στην τάξη + 10% προαιρετικό bonus

Θελω να ρωτήσω τι βαθμό έχουν τα κουίζ γιατί δεν φαίνεται να το έχετε σημειώσει στα κριτήρια βαθμολόγησης

**Comment by Angeloth1** [comment, marked as answer]
Αν δεν κάνω λάθος δεν βαθμολογείτε η επίλυση τους αλλά σου δίνουν χρήσιμες πληροφορίες για τις εργασίες, αλλά και για την συμμετοχή σου στην τάξη.
